eng/testing/tests.targets

```diff
@@ -155,7 +155,7 @@ TEST_ARCH=$(_AndroidArchitecture)
     </WriteLinesToFile>
   </Target>
 
-  <Target Name="RunTests">
+  <Target Name="RunTests" Condition="'$(MSBuildProjectName)' != 'XUnitLogChecker'">
```

Comment:
I'm confused. I thought we diagnosed that this file isn't imported for XUnitLogChecker?

Reply:
The empty "Test" target in the csproj didn't work. So I tried this instead hoping it would prevent running tests on it, but look https://dev.azure.com/dnceng-public/public/_build/results?buildId=465642&view=logs&j=fd49b2ea-3ee3-5298-d793-bdb9fd631e7e&t=26d44400-a1b8-5ed1-d719-0cc35b38251b&l=55

Reply:
Move the empty "Test" target into a Directory.Build.targets file in the same directory as the project file. Make sure that the Directory.Build.targets file imports the parent Directory.Build.targets file.